docs/tutorial.md

```diff
@@ -282,14 +281,26 @@ Added 0 files, modified 0 files, removed 1 files
 There are unresolved conflicts at these paths:
 file1    2-sided conflict
 $ cat file1
-<<<<<<<
-%%%%%%%
+<<<<<<< Conflict 1 of 1
+%%%%%%% Changes from base to side #1
 -b1
 +a
-+++++++
++++++++ Contents of side #2
 b2
->>>>>>>
-$ echo resolved > file1
+>>>>>>> Conflict 1 of 1 ends
+```
+
+`file1` now contains a [representation of the
+conflict](conflicts.md#conflict-markers). We can resolve the conflict by editing
+the file[^resolve] to remove all the conflict markers and making it into what
+the resolved version should look like. Once we've resolved the conflict, we'll
```

Comment:
> making it into what the resolved version should look like

This may be too vague. I think, with just a few more words, we can explain here more explicitly what the resolved version is expected to look like. I am thinking something along these lines:

> We can resolve the conflict by manually making the changes (as indicated by sections marked by `%%%%%%%`) in the section marked by `+++++++`, and then removing all extraneous sections and markers.